### PR TITLE
Added additional print statements to minikube start

### PR DESCRIPTION
**Pull request by aaron-prindle**
Ref: #291

### CODE REVIEW

**Comment by aaron-prindle** [comment]
The longest sections are "Starting VM..." and "Starting cluster components..."
These might require additional logging statements for users but at the same time this might make minikube start output lots of text.

Alternatively I also have a PR in my repo which has loading animations (spinner) but it seemed to have issues intermediate log messages and does not output the same on Powershell (and probably other terminals).  Also its 2000 additional lines of vendored code but we could just use the parts we need in our own code.  That is here:
https://github.com/aaron-prindle/minikube/tree/start-boot-info

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -127,28 +128,34 @@ func runStart(cmd *cobra.Command, args []string) {
 		NetworkPlugin:     viper.GetString(networkPlugin),
 		ExtraOptions:      extraOptions,
 	}
+
+	fmt.Println("SSH-ing files into VM...")
```
Comment (r2d4) [inline cmd/minikube/cmd/start.go]: I think we should leave this out.  I think we want to de-emphasize the fact its running in a VM.  Maybe "Updating cluster..."?

File: cmd/minikube/cmd/start.go [hunk 2]
```diff
@@ -127,28 +128,34 @@ func runStart(cmd *cobra.Command, args []string) {
...
 
+	fmt.Println("Connecting to cluster...")
 	kubeHost, err := host.Driver.GetURL()
 	if err != nil {
 		glog.Errorln("Error connecting to cluster: ", err)
 	}
 	kubeHost = strings.Replace(kubeHost, "tcp://", "https://", -1)
 	kubeHost = strings.Replace(kubeHost, ":2376", ":"+strconv.Itoa(constants.APIServerPort), -1)
 
+	fmt.Println("Setting up kubeconfig...")
```
Comment (r2d4) [inline cmd/minikube/cmd/start.go]: Maybe this statement could be a little lower in the conditional, saying whether it is changing the context or keeping the context.

File: cmd/minikube/cmd/start.go [hunk 3]
```diff
@@ -100,6 +100,7 @@ func runStart(cmd *cobra.Command, args []string) {
 		Downloader:          pkgutil.DefaultDownloader{},
 	}
 
+	fmt.Println("Starting VM...")
```
Comment (r2d4) [inline cmd/minikube/cmd/start.go]: nit: maybe change to "Starting up..."?  Not sure

**Pull request merged**